backend/bitboxbase/handlers/handlers.go

```diff
@@ -117,6 +138,130 @@ func (handlers *Handlers) getMiddlewareInfoHandler(r *http.Request) (interface{}
 	}, nil
 }
 
+func (handlers *Handlers) getGetHostnameHandler(r *http.Request) (interface{}, error) {
```

Comment:
Since you only use the `r` when you're posting and decoding some data, you can ignore it in all the get requests `getGetHostnameHandler(_ *http.Request)`

Reply:
Oh, also, we don't use `Get` in the namespace for getters in the app so you could just call it `HostnameHandler` and you'll have `getHostanameHandler` instead of `getGet...`